pystac_monty/sources/gdacs.py

```diff
@@ -69,25 +69,51 @@ class GDACSTransformer(MontyDataTransformer[GDACSDataSource]):
     hazard_profiles = MontyHazardProfiles()
     source_name = "gdacs"
 
-    def make_items(self) -> list[Item]:
-        items = []
-
-        # Process the main event data
-        source_event_item = self.make_source_event_item(self.data_source.data, self.data_source.source_url)
-        items.append(source_event_item)
-
-        # Process each episode
-        if self.data_source.episodes:
-            for episode_data in self.data_source.episodes:
-                episode_hazard_item = self.make_hazard_event_item(episode_data)
-
-                items.append(episode_hazard_item)
-
-                # Create impact items for this episode
-                impact_items = self.make_impact_items(episode_hazard_item, episode_data[GDACSDataSourceType.EVENT][1])
-                items.extend(impact_items)
-
-        return items
+    def get_stac_items(self) -> typing.Generator[Item, None, None]:
+        """Create STAC Items"""
+        self.transform_summary.mark_as_started()
+        self.transform_summary.increment_rows(1)
+
+        try:
+            def get_validated_event_data(item: Dict[str, Any]) -> GdacsEventDataValidator:
+                return GdacsEventDataValidator(**item)
+
+            def get_validated_geometry_data(item: Dict[Any, Any]) -> GdacsGeometryDataValidator:
+                return GdacsGeometryDataValidator(**item)
```

Comment:
We don't need these function. We can directly use GdacsEventDataValidator and GdacsGeometryDataValidator

Reply:
Done